packages/website/pages/docs/concepts/proposing.mdx

```diff
@@ -41,3 +41,15 @@ The anchor transaction is required to be the first transaction in a Taiko block
 1. Persisting `l1Height`, `l1Hash` and `l1SignalRoot` to the storage trie. These values can be used by bridges to validate cross-chain messages.
 2. Ensuring that the previous 256 block hashes that are exposed to the EVM are correct.
 3. Calculating the EIP-1559 `basefee` which will be used on L2.
+
+## How does the proposer earn Ether
+
+When the block is proposed the taiko-client will send the `L2_SUGGESTED_FEE_RECIPIENT` in the proposeBlock() input data. The TaikoL1 contract sets this value in the L2 block metadata which is stored on L1.
```

Comment:
btw just fyi, in A6's protocol, i see proposers can not customize the L2 `block.coinbase` anymore: https://github.com/taikoxyz/taiko-mono/blob/contestable-zkrollup/packages/protocol/contracts/L1/libs/LibProposing.sol#L145

Reply:
Right. @d1onys1us maybe new documentation improvements should beased on the A6 branch -- https://github.com/taikoxyz/taiko-mono/tree/contestable-zkrollup

Reply:
@dantaik @davidtaikocha why are we removing the ability to set the `l2SuggestedFeeRecipient`? this may remove some flexibility for L2 block builders to create deals with L1 validators, such as if the L2 block builder wants to receive the tips and only send some other incentive at the bottom of the L1 block or something.